TailLight/TailLight.vcxproj

```diff
@@ -82,12 +83,13 @@ copy "$(WDKBinRoot_x64)\certmgr.exe" $(PackageDir)</Command>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <Link>
-      <AdditionalDependencies>%(AdditionalDependencies);$(DDK_LIB_PATH)\hidparse.lib</AdditionalDependencies>
+      <AdditionalDependencies>%(AdditionalDependencies);$(DDK_LIB_PATH)\hidparse.lib;$(DDK_LIB_PATH)\ntstrsafe.lib</AdditionalDependencies>
```

Comment:
Are you sure that you need to link to `ntstrsafe.lib` here` If yes, then why are you only introducing linking for "debug" builds, and not "release"? This seems very odd to me.

Reply:
Verified that ntstrsafe.lib does not need to be explicitly brought in.